libevmasm/RuleList.h

```diff
@@ -163,16 +199,13 @@ std::vector<SimplificationRule<Pattern>> simplificationRuleListPart1(
 }
 
 
-// This part of simplificationRuleList below was split out to prevent
```

Comment:
Maybe move this to the top saying the rules are split due to JS?

Reply:
There was a copy of this at the top already - I just rephrased it slightly now.